src/sentry/static/sentry/app/views/organizationDiscover/sidebar/orderBy.jsx

```diff
@@ -0,0 +1,99 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SelectControl from 'app/components/forms/selectControl';
+import {t} from 'app/locale';
+import styled from 'react-emotion';
+import {Flex} from 'grid-emotion';
+
+import {PlaceholderText, SidebarLabel} from '../styles';
+
+export default class OrderBy extends React.Component {
+  static propTypes = {
+    value: PropTypes.string.isRequired,
+    columns: PropTypes.array.isRequired,
+    onChange: PropTypes.func.isRequired,
+    disabled: PropTypes.bool,
+  };
+
+  updateField(field) {
+    const orderby = this.getInternal(this.props.value);
+    orderby.field = field;
+    this.props.onChange(this.getExternal(orderby));
+  }
+
+  updateDirection(direction) {
+    const orderby = this.getInternal(this.props.value);
+    orderby.direction = direction;
+    this.props.onChange(this.getExternal(orderby));
+  }
+
+  /**
+   * @param {Object} i.e. {direction: 'asc', field: 'id'}
```

Comment:
can you make this:
```
@param {Object} value Object containing orderby information
@param {String} value.direction Either 'asc' or 'desc'
@param {String} value.field Field being ordered
@returns {String}
```
if an example is required, it can go below the @param and @returns statements